crates/miden-testing/src/tx_context/context.rs

```diff
@@ -138,6 +141,11 @@ impl TransactionContext {
         // Note that native and foreign account's code are already loaded by the
         // TransactionContextBuilder.
         self.mast_store.insert(TransactionKernel::library().mast_forest().clone());
+        self.mast_store.insert(miden_lib::StdLibrary::default().mast_forest().clone());
+        self.mast_store.insert(miden_lib::MidenLib::default().mast_forest().clone());
+        self.mast_store
+            .insert(AccountCode::mock_account_library().mast_forest().clone());
+        self.mast_store.insert(AccountCode::mock_faucet_library().mast_forest().clone());
         self.mast_store.insert(program.mast_forest().clone());
```

Comment:
Why did this work without the stdlib and miden lib before?

Do we really have to insert the mock account and faucet libraries? These should only be relevant when executed against a mock account, in which case the account's code itself will bring these libraries and so no dynamic lookup should be required:

https://github.com/0xMiden/miden-base/blob/a5b4b5bf65ee5153a7412873914e6d8b4f04288b/crates/miden-lib/src/testing/account_component/mock_account_component.rs#L55-L61